sdk/translation/azure-ai-translation-text/tests/test_break_sentence.py

```diff
@@ -21,7 +21,7 @@ def test_autodetect(self, **kwargs):
         response = client.find_sentence_boundaries(request_body=input_text_elements)
         assert response is not None
         assert response[0].detected_language.language == "en"
-        assert response[0].detected_language.score == 1
+        assert response[0].detected_language.score == 0.98
```

Comment:
If this score isnt 100% reliable you could just check if it's >= 0.9

Reply:
Good idea, done.